_pages/for-users.md

```diff
@@ -14,7 +14,7 @@ Here's a quick tour of what you'll see in the demo:
 
   <div class="card">
 	  <div class="card-header">
-	    <img style="height: 100%; width: 100%;" src="/assets/images/demo_login.png" />
+	    <img style="height: 100%; width: 100%;" src="/assets/images/demo_login.png" alt="screenshot of the ChRIS web application showing a demo login"/>
```

Comment:
Another nitpick: every other alt has its first letter capitalized, so let's do the same here.

Reply:
Sure, okay!